### DISCUSSION TITLE
Which animal can be found on the Toblerone logo?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#97] [Galaxy Trivia] [answered]
**Hint:** 🐻

**Comment by Tynab** [comment, marked as answer]
The animal that can be found on the Toblerone logo is a bear. Specifically, the Toblerone logo features a stylized image of a bear standing on its hind legs, with its front paws raised up. The bear is a reference to the city of Bern, Switzerland, which is sometimes called the "City of Bears" and has a bear featured in its coat of arms. The Toblerone brand was created in Bern in 1908.
